DOCS-1781: Adds close_window description to API (#1320)
MultiSafepay/docs: content/api/orders/post_order.md

Customer’s provided email address. Used to send Second Chance emails and in fraud checks.

----------------

__close_window__ | bool (optional)

> Options: true, false. Set to true if you want to display the MultiSafepay payment page in a new window and want to close it automatically after the payment process. 

Reviewer: ```suggestion
Options: `True`, `False`. To display the MultiSafepay payment page in a new window that automatically closes after the payment is completed, set to `True`. 
```

> Options: true, false. Set to true if you want to display the MultiSafepay payment page in a new window and want to close it automatically after the payment process. 

Reviewer: ```suggestion
Options: `True`, `False`. To display the MultiSafepay payment page in a new window that automatically closes after the payment is completed, set to `True`. 
```
